css/templates/lune/components/_sidebar.css

```diff
@@ -0,0 +1,58 @@
+@import "../page-vars";
+
+.ampstart-sidebar {
+  width: 80vw;
+  display: flex;
+  align-items: center;
+}
+
+/* Position the close button at the top of the sidebar */
+/* Make it the same height as the header in order to align the close and hamburger icons */
+.ampstart-sidebar-header {
+  position: absolute;
+  top: 0;
+  height: var(--layout-header-height-narrow);
```

Comment:
`--navbar-height` already exists, override this value instead?

For wide navbar, I created a new css var `--wide-navbar-height` and overrode where neccessary. Like so:
```
@media (--breakpoint-md) {
  .ampstart-headerbar + *:not(amp-sidebar),
  .ampstart-headerbar + amp-sidebar + * {
    margin-top: var(--wide-navbar-height);
  }
}
```